src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Assembly.cs

```diff
@@ -563,7 +563,7 @@ class Program
 {
     void M(Test x) {}
 }
-", options: TestOptions.ReleaseDll, references: new[] { MscorlibRef_v4_0_30316_17626, hash_module });
+", options: TestOptions.ReleaseDll, references: new[] { MscorlibRef_v45, hash_module });
```

Comment:
I find it very confusing to go by the file version when implementing tests. It is reasonable to name actual blob resources like that, but from test writer prospective _45 is a much friendlier name